storage/mongodb/files/scripts/test-mongo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+readonly NAMESPACE=${1}
+readonly ITERATION=${2}
+
```

Comment:
In my opinion this will work fine, but for future works I propose we make parameters for ```threads``` too

Reply:
Also addition to this is below 
- having THREADS as input parameter will help to run different values for threads 
yctb produces very big output, but out of its output the most interesting part is 
```
[OVERALL], RunTime(ms), 26078.0
[OVERALL], Throughput(ops/sec), 38346.498964644525
[TOTAL_GCS_PS_Scavenge], Count, 33.0
[TOTAL_GC_TIME_PS_Scavenge], Time(ms), 105.0
[TOTAL_GC_TIME_%_PS_Scavenge], Time(%), 0.4026382391287675
[TOTAL_GCS_PS_MarkSweep], Count, 0.0
[TOTAL_GC_TIME_PS_MarkSweep], Time(ms), 0.0
``` 
especially 
```
[OVERALL], Throughput(ops/sec), 38346.498964644525
``` 
and this info is necessary  to see what is throughput for particular test 

an idea how to modify this script could be 
``` 
for i in $(seq 1 ${ITERATION});
do
  echo "iteration: ${i}"
  oc exec $(oc get pod -n ${NAMESPACE} | grep mongodb | awk '{print $1}') -- scl enable rh-mongodb32 -- mongo -u redhat -p redhat ${MONGODB_IP}:27017/testdb --eval "db.usertable.remove({})"
  oc exec $(oc get pod -n ${NAMESPACE} | grep ycsb | awk '{print $1}') -- ./bin/ycsb load mongodb -s -threads ${THREADS} -P workloads/workload_template -p mongodb.url=mongodb://redhat:redhat@${MONGODB_IP}:27017/testdb | egrep 'RunTime|Throughput' | cut -d',' -f2,3 > result_run_$i.txt
  grep Throughput result_run_$i.txt  | cut -d',' -f2 >> mongodb_result_threads_${THREADS}.txt 
done
``` 
Now, there is open question regarding where to write these data and we have two options 
- a) if pbench is not used then just creating directory where to send these results will be enought 
- b) if pbench is used these needs to be send to /var/lib/pbench-agent/pbench-user-benchmark-xxxx-timestamp
and this is easy to make to write at ```pbench``` location if pbench exported variable is used ( once pbench-user-benchmark is started it will be there ) $benchmark_run_dir - example how  is used can be seen at below link  https://github.com/openshift/svt/blob/master/storage/postgresql/pgbench_test.sh#L145-L151

Reply:
Done.
threads is not a var in playbook.
keep the original result.txt file intact and grep the throughput lines into another file throughput_result.txt, eg, [throughput_result.txt](http://pbench.perf.lab.eng.bos.redhat.com/results/EC2::ip-172-31-26-32/pbench-user-benchmark_mongodb_storage_test_gp2_2018.05.03T16.27.02/1/reference-result/throughput_result.txt).